raster-foundry/raster-foundry
# Add classes to campaign export label STAC item and label data

## Overview

This PR fixes a bug in campaign STAC exports so that:
- the label data has class name information
- the label stac time has class names

### Checklist

- ~Swagger specification updated~
- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~
- ~Any new endpoints have scope validation and are included in the integration test csv~

## Testing Instructions

- Assemble batch jar, rebuild batch service, spin up servers
- Connect local Groundwork to your local RF servers
- In a campaign with class definition and labels, start an export, and grab the export ID
- Open a batch bash shell while servers are running, run `java -cp /opt/raster-foundry/jars/batch-assembly.jar com.rasterfoundry.batch.Main write_stac_catalog <export id>`
- On the Groundwork frontend, download the export,
- Make sure the label data has label name info. Make sure the label stac item has label class info.

Closes https://github.com/raster-foundry/raster-foundry/issues/5567
